pages/common/rdfind.md

```diff
@@ -0,0 +1,19 @@
+# rdfind
+
+> Finds duplicate files and gets rid of them.
+
+- Identify all duplicates in a given directory and output summary:
+
+`rdfind -dryrun true {{path/to/directory}}`
+
+- Replace all duplicates with hardlinks:
+
+`rdfind -makehardlinks true {{path/to/directory}}`
+
+- Replace all duplicates with symlinks/soft links:
+
+`rdfind -makesymlinks true {{path/to/directory}}`
+
+- Delete all duplicates, do not ignore empty files:
```

Comment:
Perhaps ...`duplicates, and do not ignore`.... would read better?

Reply:
Valid point, i like that one better; Thank you

Reply:
I think the `,` is superfluous. We should probably drop it.

Reply:
Agreed, removed it;